psykorebase/cli.py

```diff
@@ -62,11 +88,14 @@ def __init__(self, repository_path, primary_branch, secondary_branch=None):
 
     def rebase(self):
         """Actually rebase."""
-        self.checkout(self.primary_branch)
-        self.checkout(self.target_branch, create=True)
-        self.merge(self.secondary_branch,
-                   message='Psycho-rebased branch %s on top of %s'
-                           % (self.secondary_branch, self.primary_branch))
+        if not self.rebase_continue:
+            self.checkout(self.primary_branch)
+            self.checkout(self.target_branch, create=True)
+            self.merge(self.secondary_branch, no_ff=self.no_ff,
```

Comment:
What about catching exceptions here? In order to add a message that says something like "Error, resolve conflicts then run psykorebase --continue" instead of a traceback.